dss/config.py

```diff
@@ -204,6 +206,15 @@ def get_blobstore_handle(replica: "Replica") -> BlobStore:
     def get_hcablobstore_handle(replica: "Replica") -> HCABlobStore:
         return replica.hcablobstore_class(Config.get_blobstore_handle(replica))
 
+    @staticmethod
+    @functools.lru_cache()
+    def get_flashflood_handle(prefix: str, confirm_writes: bool=False) -> flashflood.FlashFlood:
```

Comment:
This is a good pattern  👍  

(makes changing the constructor method signature easier too)